scompose/project/project.py

```diff
@@ -148,15 +145,59 @@ def get_already_running(self):
     def load(self):
         """load a singularity-compose.yml recipe, and validate it."""
 
-        if not os.path.exists(self.filename):
-            bot.error("%s does not exist." % self.filename)
-            sys.exit(1)
-
         try:
-            self.config = read_yaml(self.filename, quiet=True)
+            yaml_files = []
+
+            for f in self.filename:
+                # ensure file exists
+                if not os.path.exists(f):
+                    bot.error("%s does not exist." % f)
```

Comment:
Please move the multiple load of yaml files outside of the project.py - there should ideally be a separate utils function or set of config functions (some config.py or similar) that takes the list. and returns the single config here. Also - self.filename implies one file, and there should either be a self.filenames or self.filename to match the arity.